config.ml

```diff
@@ -53,6 +53,14 @@ let disk =
 
 (* Command-line options *)
 
+let root_k =
+  let doc = Key.Arg.info ~doc:"Blog URL" ["root"] in
+  Key.(create "root" Arg.(opt string "http://localhost" doc))
```

Comment:
by default, canopy listens on port 8080 -- shouldn't this be reflected here in the url (and have the default value be `http://localhost:8080`).